src/components/vote/VoteList.js

```diff
@@ -23,32 +24,36 @@ class VoteList extends Component {
   }
 
   getUnvotedAwards() {
+    const { currentUser } = this.props;
     const { awards } = this.state;
-    return awards.filter((award) => {
-      const { currentUser } = this.props;
-      return !currentUser.votes
-        || !currentUser.votes.find(userVoted => userVoted === award.id);
-    });
+
+    return awards.filter(award => !currentUser.votes
+      || !currentUser.votes.find(userVoted => userVoted === award.id));
   }
 
   submitVote(awardId, voteObject) {
-    this.db.collection(`cohorts/D32/awards/${awardId}/votes`).doc().set(voteObject);
+    const { currentUser, updateCurrentUserVote } = this.props;
+    this.db.collection(`cohorts/${currentUser.cohortId}/awards/${awardId}/votes`).doc().set(voteObject);
+    updateCurrentUserVote(awardId);
```

Comment:
You might call `updateCurrentUserVote` inside the callback function for the `set` call above. This is in case the call to the database fails, the user will want to be able to try again.

```javascript
this.db.collection(`cohorts/${currentUser.cohortId}/awards/${awardId}/votes`).doc().set(voteObject)
    .then(() => updateCurrentUserVote(awardId));
```

Reply:
That makes sense. I refactored as suggested and tested.